Lib/multiprocessing/process.py

```diff
@@ -336,6 +335,13 @@ def _bootstrap(self, parent_sentinel=None):
 
         return exitcode
 
+    @staticmethod
+    def _bootstrap_util():
```

Comment:
The name is a bit confusing, call it `_after_fork` or something similar?

Reply:
I've chaned all `_bootstrap_util ` to `_after_fork`.